src/sentry/utils/strings.py

```diff
@@ -93,7 +93,7 @@ def soft_break(value, length, process=lambda chunk: chunk):
     zero-width spaces after common delimeters, as well as soft-hyphenating long
     identifiers.
     """
-    delimiters = re.compile(ur'([{}]+)'.format(''.join(map(re.escape, ',.$:/+@!?()<>[]{}'))))
+    delimiters = re.compile(r'([{}]+)'.format(''.join(map(re.escape, ',.$:/+@!?()<>[]{}'))))
```

Comment:
How do you do this then to use a unicode instead of bytes? I ultimately want `unicode.format` here instead of `str.format`.

Reply:
I guess that makes sense, just gross. I know there's no difference in py3, but it's a big difference in py2. Can we change this PR to wrap in `text_type` then? And I'll merge.

Reply:
Wrapped in __six.text_type()__ as requested.